### DISCUSSION TITLE
Panic when primary key of active model is not set during update

### DISCUSSION THREAD

**Discussion by billy1624** [SeaQL/sea-orm#98] [Ideas]
When a primary key is not included in an active model, sea-orm panics. Could this be handled in a better way?
https://github.com/SeaQL/sea-orm/blob/master/src/query/update.rs#L95

```rust
let active_model = cake::ActiveModel {
    // Don't include ID ...
    ..Default::default()
};

let products = Cake::update(active_model).exec(&db).await?;
```

@Acidic9 thanks for raising this issue

**Comment by tyt2y3** [comment]
merge to https://github.com/SeaQL/sea-orm/discussions/96
